alembic/versions/2021_07_29_18_00_833ba1ee8c72.py

```diff
@@ -0,0 +1,27 @@
+"""Add thumbnail column for mod table
+
+Revision ID: 833ba1ee8c72
+Revises: 17fbd4ff8193
+Create Date: 2021-07-29 18:50:08.327728+00:00
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '833ba1ee8c72'
+down_revision = '17fbd4ff8193'
+
+from alembic import op
+import sqlalchemy as sa
+
+
+def upgrade() -> None:
+    op.execute("UPDATE mod SET background = trim(LEADING '/content/' FROM background) "
+               "WHERE background IS NOT NULL AND background != ''")
+    op.execute("UPDATE \"user\" SET \"backgroundMedia\" = trim(LEADING '/content/' FROM \"backgroundMedia\") "
```

Comment:
Could we use `'` instead of `\"` as in the previous line? Slightly easier to read that way, I'd think.

Reply:
No, double quotes and single quotes have different meanings in SQL. Double quotes are for column and table names, single quotes for strings. Replacing them with single quotes doesn't work, and replacing the single quotes with double quotes doesn't work either, it throws syntax errors.

Reply:
We need the double quotes for `user`, because `user` is also an SQL instruction. And when double-quoting the table name, you apparently need to double quote all the column names as well.

Reply:
I guess I could try triple quotes on the outside? Not sure if they're handled specially by Python though.

Reply:
OK, that's fine, this isn't a big problem, just a question I had while reviewing. Never mind!

Reply:
Changed to triple quotes regardless :laughing: 
I do agree that escaped strings are hard to read, and it does appear to work just fine.